lib/rspec/rails/configuration.rb

```diff
@@ -71,6 +71,11 @@ def self.initialize_configuration(config) # rubocop:disable Metrics/MethodLength
       config.add_setting :fixture_path
       config.include RSpec::Rails::FixtureSupport, :use_fixtures
 
+      # Prevent test from failing on undefined method used in ActiveSupport testing helpers
+      if ::Rails::VERSION::STRING.to_f >= 6.0
+        config.include ActiveSupport::Testing::Assertions
+      end
```

Comment:
I think this implementation needs to go in `lib/rspec/rails/adapters.rb` or `lib/rspec/rails/example/rails_example_group.rb `

Reply:
I tried this initially but was not able to make it work.

For `lib/rspec/rails/example/rails_example_group.rb` with:
```diff
--- a/lib/rspec/rails/example/rails_example_group.rb
+++ b/lib/rspec/rails/example/rails_example_group.rb
@@ -12,6 +12,11 @@ module RSpec
       include RSpec::Rails::MinitestLifecycleAdapter
       include RSpec::Rails::MinitestAssertionAdapter
       include RSpec::Rails::FixtureSupport
+
+      # Prevent test from failing on undefined method used in ActiveSupport testing helpers
+      if ::Rails::VERSION::STRING.to_f >= 6.0
+        include ActiveSupport::Testing::Assertions
+      end
```

From a Rails app that use rspec-rails with a test that call `assert_nothing_raised`, I get:
```
NameError:
  uninitialized constant ActiveSupport::Testing::Assertion
```

For `lib/rspec/rails/adapters.rb` this is pretty complicated inside. I tried to put in in multiple places but I am having error like from my Rails app:

```
NameError:
  uninitialized constant ActiveSupport::Testing
```

The other place were we properly load an active support testing class is in 
https://github.com/rspec/rspec-rails/blob/rails-6-1-dev/lib/rspec/rails/fixture_file_upload_support.rb#L42 
and then it is loaded in configuration https://github.com/rspec/rspec-rails/blob/0e5d60611d719b0d201993b22ba42d252f01d5f3/lib/rspec/rails/configuration.rb#L94

I am stuck to provide you another solution ATM. 😬